benchmark/src/lib.rs

```diff
@@ -325,9 +339,11 @@ impl Benchmarker {
         let children: Vec<thread::JoinHandle<_>> = txn_req_chunks
             .zip(self.clients.iter().cycle())
             .map(|(chunk, client)| {
-                let local_chunk = Vec::from(chunk);
                 let local_client = Arc::clone(client);
+                let mut local_chunk = Vec::from(chunk);
                 let stagger_range_ms = self.stagger_range_ms;
+                let mut local_pattern = *pattern;
```

Comment:
If this part still stays after the changes to how `pattern` is implemented -- it will be clearer if you directly clone vs deref.

Reply:
Yes. Now we explicitly **clone** `pattern` into each thread.